3.4/deployment-active-failover-using-the-starter.md

```diff
@@ -17,28 +31,29 @@ option of the _Starter_. This will start all servers within the context of a sin
 starter process:
 
 ```bash
-arangodb --starter.local --starter.mode=activefailover --starter.data-dir=./localdata
+arangodb --starter.local --starter.mode=activefailover --starter.data-dir=./localdata --auth.jwt-secret=/etc/arangodb.secret
+```
+
+Please adapt the path to your _secret_ file accordingly.
 ```
```

Comment:
These quotes seem unintended and break the formatting.
![image](https://user-images.githubusercontent.com/9152640/67500329-cdc60980-f682-11e9-9561-e9ec8d6db037.png)

Reply:
thanks - done